static/bidder-info/bluesea.yaml

```diff
@@ -0,0 +1,11 @@
+endpoint: "https://bid.bluevoox.com/rtb/prebid"
+maintainer:
+  email: prebid@blueseasx.com
```

Comment:
@blueseasx,  an email is sent by Prebid team to verify maintainer's email address. Please respond to email with "received" message to complete verification

Reply:
@blueseasx, `prebid@blueseasx.com` is not reachable

Reply:
hi @onkarvhanumante   sorry to apply incorrect filter rules on this email. It is resolved, please try again.

Reply:
@blueseasx  verification mail is being sent.  Please respond to email with "received" message to complete verification